framework/doc/content/source/libtorch/userobjects/TorchScriptUserObject.md

```diff
@@ -0,0 +1,20 @@
+# TorchScriptUserObject
+
+!syntax description /UserObjects/TorchScriptUserObject
+
+## Description
+
+This user object loads a torch module that has been exported from
+pytorch in a torch script format. The user object loads the modules
+when the user object is execute which can be controlled using the
+`execute_on` flag.
```

Comment:
It'll be nice to link to an external documentation explaining what "torch script" is.